### PR TITLE
Achadha/associate services with category

### PR DESCRIPTION
**Pull request by achadha**

### CODE REVIEW

File: app/components/App.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import Navigation from './Navigation';
 import CategoryPage from './Find/FindPage';
 import ResourcesTable from './Search/ResourcesTable';
+import 'react-select/dist/react-select.css';
```
Comment (achadha) [inline app/components/App.js]: I assume I can just add something to the webpack config to import this CSS?

Comment (richardxia) [inline app/components/App.js]: It looks like this works fine. I found this thread that shows how you can do it with the CSS modules-loader, but we're not using it: https://github.com/JedWatson/react-select/issues/1324#issuecomment-256591621

File: app/components/Admin/ProposedService.js [hunk 2]
```diff
@@ -86,6 +95,16 @@ class ProposedService extends React.Component {
     }
     return scheduleOutput;
   }
+
+  renderCategoryFields() {
+    return (
+      <div key={"cat"} className="request-entry">
+        <p className="request-cell name">Categories</p>
+        <textarea readOnly className="request-cell value" value={this.state.categories} />
```
Comment (richardxia) [inline app/components/Admin/ProposedService.js]: So this is readonly? That's probably fine for now, but we should let the data team know that they won't be able to edit the proposed categories.

File: app/components/Edit/CategoriesDropdown.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+import * as dataService from '../../utils/DataService';
+
+class CategoriesDropdown extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedValue: '',
```
Comment (richardxia) [inline app/components/Edit/CategoriesDropdown.js]: Should this be renamed to `selectedValues` and initialized to `[]`? Looks like when you update it, it gets set to an array of values.

File: app/components/Admin/ProposedService.js [hunk 4]
```diff
@@ -86,6 +95,16 @@ class ProposedService extends React.Component {
     }
     return scheduleOutput;
   }
+
+  renderCategoryFields() {
+    return (
+      <div key={"cat"} className="request-entry">
```
Comment (richardxia) [inline app/components/Admin/ProposedService.js]: I don't really know React that well, but do you need a `key` if this isn't really a list of stuff?

Comment (achadha) [inline app/components/Admin/ProposedService.js]: Oops you don't. Bad job copy-pasting and then editing lol

File: app/components/Admin/ProposedService.js (hunk 2 above)
Comment (achadha) [inline app/components/Admin/ProposedService.js]: Yeah, after talking with joe we realized it was complicated. All of the other fields are string values, but here, you want to edit the object. As an enhancement, I can write the stuff to map the edited strings to the category objects.

**Pull request merged**